testing/percona-xtradb-cluster/APKBUILD

```diff
@@ -0,0 +1,226 @@
+    install -Dm750 -o mysql -d "$subpkgdir"/usr/lib/mysql || return 1
+    install -Dm750 -o mysql -d "$subpkgdir"/run/mysqld || return 1
+    mkdir -p "$subpkgdir"/usr/share/mysql \
+        "$subpkgdir"/etc \
+        "$subpkgdir"/usr/lib/mysql/plugin \
+        || return 1
+    mv "$pkgdir"/etc/mysql "$subpkgdir"/etc/ || return 1
+    local lang="charsets danish english french greek italian korean norwegian-ny
+        portuguese russian slovak swedish czech dutch estonian german
+        hungarian japanese norwegian polish romanian serbian spanish
+        ukrainian bulgarian"
+    for l in $lang; do
+        mv "$pkgdir"/usr/share/mysql/$l \
+            "$subpkgdir"/usr/share/mysql/ || return 1
+    done
+}
+
+_client_libs() {
+    pkgdesc="Percona XtraDB Cluster client library"
+    replaces="percona-xtradb-cluster libmysqlclient"
+    depends="$pkgname-common"
+    mkdir -p "$subpkgdir"/usr/lib \
+        "$subpkgdir"/usr/share/mysql \
+        || return 1
+    mv "$pkgdir"/usr/lib/libperconaserverclient.so* \
+        "$subpkgdir"/usr/lib/ \
+        || return 1
+}
+
+mytest() {
+    pkgdesc="The test suite distributed with Percona XtraDB Cluster"
+    mkdir -p "$subpkgdir"/usr/bin || return 1
+    mv "$pkgdir"/usr/bin/mysql_client_test \
+        "$pkgdir"/usr/mysql-test \
+        "$pkgdir"/usr/bin/my_safe_process \
+        "$subpkgdir"/usr/bin/ \
+        || return 1
+}
+
+client() {
+    pkgdesc="client for the Percona XtraDB Cluster database"
+    depends="$pkgname-common perl-dbi"
+    replaces="mysql-client mariadb-client"
+    mkdir -p "$subpkgdir"/usr/bin/ || return 1
+    local bins="myisam_ftdump mysql mysqladmin
+        mysqlcheck mysqldump mysqldumpslow
+        mysqlimport mysqlshow mysqlslap mysql_config_editor"
+    for i in $bins; do
+        mv "$pkgdir"/usr/bin/${i} "$subpkgdir"/usr/bin/ || return 1
+    done
+}
+
+server() {
+    pkgdesc="server for the Percona XtraDB Cluster database"
+    depends="$pkgname-common $pkgname-galera percona-xtrabackup socat iproute2
+        procps findutils coreutils tzdata bash perl-dbd-mysql"
+    replaces="mariadb"
+    mkdir -p "$subpkgdir"/usr/lib/mysql/plugin || return 1
+    mv "$pkgdir"/usr/lib/mysql/plugin/*.so \
+        "$subpkgdir"/usr/lib/mysql/plugin/ || return 1
+    local bins="mysqld clustercheck pyclustercheck innochecksum my_print_defaults
+        myisamchk myisamlog myisampack mysql_install_db mysql_secure_installation
+        mysql_tzinfo_to_sql mysql_upgrade mysql_plugin mysqlbinlog mysqld_multi
+        mysqld_safe mysqltest perror replace resolve_stack_dump resolveip
+        wsrep_sst_common wsrep_sst_mysqldump wsrep_sst_xtrabackup-v2 wsrep_sst_rsync
+        mysql_ssl_rsa_setup"
+    mkdir -p "$subpkgdir"/usr/bin/ || return 1
+    for i in $bins; do
+        mv "$pkgdir"/usr/bin/${i} "$subpkgdir"/usr/bin/ || return 1
+    done
+}
+
+md5sums="58f8524be1e25cc740cfa009d3e7c09c  Percona-XtraDB-Cluster-5.7.16-27.19.tar.gz
```

Comment:
Do `abuild checksum` with new abuild as `md5sums` and `sha256sums` are no longer needed or generated.